### PR TITLE
HW#2 push

### PR DESCRIPTION
**Pull request by dstad418**
Please review!

### CODE REVIEW

**Review by SayleeMJ — APPROVED** [review]
The code looks great. I really your code for changing the background color and the naming convention with RGBA in the randomColor() function.

File: hw2/03-count.js [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+      let highlightedText = "";
+      words.forEach((word) => {
+        const regex = new RegExp(`\\b${searchWord}\\b`, "i");
+        if (regex.test(word)) {
+          highlightedText += `<span class="highlight">${word}</span> `;
+        } 
+        else {
+          highlightedText += `${word} `;
+        }
+      });
```
Comment (lebenpdx) [inline hw2/03-count.js]: At the moment, when you clear out the wordInput, it highlights the whole passage. I would nest this in a if statement to check if the wordInput is empty or not. I think you would also need to keep an original version of the passage to restore in this case as well.

File: hw2/04-background.js [hunk 2]
```diff
@@ -1 +1,53 @@
...
+    function startBackgroundChange() {
+      setBackgroundColor();
+      timer = setInterval(setBackgroundColor, interval);
+    }
+  
+    function stopBackgroundChange() {
+      clearInterval(timer);
+    }
+  
+    toggleButton.addEventListener("click", function () {
```
Comment (lebenpdx) [inline hw2/04-background.js]: I think the background color is supposed to start on load. A quick way I did this was to simulate a click by using toggleButton.click().

**Review by lebenpdx — COMMENTED** [review]
I have finished my review.

File: hw2/04-background.js (hunk 2 above)
Comment (dstad418) [reply hw2/04-background.js]: Completed! Thanks for your review!

File: hw2/03-count.js (hunk 1 above)
Comment (dstad418) [reply hw2/03-count.js]: Completed! Thanks for your review!

**Comment by dstad418** [comment]
Completed all revisions per PR review.

**Comment by mattsloe** [comment]
Hi, all of the webpage exercises seem to be working well. Only a few suggestions vis a vis the airbnb style guide. 

* Airbnb style guide suggestions
  * single quotes are preferred for strings
  * style guide prefers arrow notation callback functions `(event) => {}` instead of `function(event) {}`

**Pull request merged**